src/components/result/utils/ResultConstants.ts

```diff
@@ -0,0 +1,7 @@
+const RESULT_VALUES = {
+  TITLE: "title",
+  DESCRIPTION: "description",
+  IMAGE: "image"
+}
+
+export {RESULT_VALUES};
```

Comment:
Are we using this class? If we are using it what is the use-case? If you clarify it would be very good 😃 @yasincaliskan

Reply:
Actually it was necessary for different structure but changed. I will remove them.